ckanext/dcatapchharvest/tests/test_dcatap_ch_parse.py

```diff
@@ -101,6 +103,28 @@ def test_dataset_all_fields(self):
         see_also = dataset['see_alsos'][0]
         eq_(see_also['dataset_identifier'], u'4682791@bundesamt-fur-statistik-bfs')
 
+        relations = sorted(dataset["relations"], key=lambda relation: relation['url'])
+
+        # Relations - only one label given, no language specified
+        eq_(relations[0]['url'], "https://www.admin.ch/opc/de/classified-compilation/19920252/index.html")
+        for lang in self.languages:
+            eq_(relations[0]['label'][lang], 'legal_basis')
+
+        # Relations - multilingual labels
+        eq_(relations[1]['url'], "https://www.example.org/aaa")
+        for lang in self.languages:
+            eq_(relations[1]['label'][lang], 'Text for label ' + lang.upper())
+
+        # Relations - no label given
+        eq_(relations[2]['url'], "https://www.example.org/bbb")
+        for lang in self.languages:
+            eq_(relations[2]['label'][lang], "https://www.example.org/bbb")
+
+        # Relations - label given, language specified but not German
+        eq_(relations[3]['url'], "https://www.example.org/ccc")
+        for lang in self.languages:
+            eq_(relations[3]['label'][lang], 'Text for label IT')
```

Comment:
should we specify to use "it" lang for this test?

Reply:
Not sure what you mean here?

Reply:
If you mean this:
```suggestion
        eq_(relations[3]['url'], "https://www.example.org/ccc")
        eq_(relations[3]['label']['it'], 'Text for label IT')
```

then we definitely shouldn't, because this test is making sure that if we only get a label in one language, we use it for all the languages (instead of using the url for the other languages).

Reply:
ahhh, now i see

Reply:
I think I will add a quick comment to the test, since it wasn't clear :)